archetype-common/src/test/java/org/apache/maven/archetype/old/ArchetypeTest.java

```diff
@@ -61,19 +61,11 @@
 public class ArchetypeTest
     extends PlexusTestCase
 {
-    private static final String XML_VERSION_1_0 =
-            "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n";
-    private static final String PROJECT_ELEMENT = "<project " +
-            "xmlns=\"http://maven.apache.org/POM/4.0.0\" " +
-            "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" " +
-            "xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 " +
-            "http://maven.apache.org/xsd/maven-4.0.0.xsd\">\n";
-    private static final String PROJECT_ELEMENT_END = "</project>";
-    private static final String MODEL_VERSION = "  <modelVersion>4.0.0</modelVersion>\n";
-    private static final String PACKAGING = "  <packaging>pom</packaging>\n";
 
     private OldArchetype archetype;
 
+    private StringWriter out;
```

Comment:
initialize this here

Reply:
ping

Reply:
It is initialized in the setup method. If I init it here it would be done twice. If I remove it from the setup the tests would share state.

Or do I miss something here?

Btw if any maintainer _really_ wants it to be changes, please fell free to do so. Whatever works best, I am fine with it.

Reply:
Yes, you're missing something. Tests do not share instance variables. Every test is run with a new object.

Reply:
Done.